numeric.c

```diff
@@ -2215,7 +2215,7 @@ rb_int_truncate(VALUE num, int ndigits)
 
 /*
  *  call-seq:
- *     float.round([ndigits])  ->  integer or float
+ *     float.round([ndigits] [, half: symbol])  ->  integer or float
```

Comment:
As mentioned in my PR description, I am not entirely sure if this is the correct way to specify an optional keyword argument in the function's contract.